test/utils_shared/test_utils.py

```diff
@@ -393,6 +394,15 @@ def assert_exception_correct(got: Exception, expected: Exception):
     assert type(got) == type(expected)
 
 
+def assert_close_to_now(time_):
```

Comment:
slightly odd variable name -- I'd expect `_time` or perhaps `time_to_test`. The hanging underscore is unexpected!

Reply:
Suffixing a keyword with an underscore is actually a standard convention in python:
https://www.datacamp.com/community/tutorials/role-underscore-python#STU
https://www.geeksforgeeks.org/underscore-_-python/

Reply:
Mostly I have seen it used, as it is in this case, to distinguish the variable from the corresponding standard library object or keyword.